methods/array_ops/src/pg_gp/array_ops.c

```diff
@@ -702,8 +702,14 @@ array_sub(PG_FUNCTION_ARGS){
 PG_FUNCTION_INFO_V1(array_mult);
 Datum
 array_mult(PG_FUNCTION_ARGS){
-    if (PG_ARGISNULL(0)) { PG_RETURN_NULL(); }
-    if (PG_ARGISNULL(1)) { PG_RETURN_NULL(); }
+    if (PG_ARGISNULL(0) && PG_ARGISNULL(1)) { PG_RETURN_NULL(); }
```

Comment:
Well, I agree that NULL being treated as identity is a little weird. In that sense it is not consistent with array_add